docs/tutorials/ring_background.ipynb

```diff
@@ -385,6 +393,116 @@
     "print(f\"Fit results: mu = {mu:.2f}, std = {std:.2f}\")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Modeling and fitting\n",
+    "\n",
+    "Once we have a good choice of exclusion regions, we are often interested in a morphological modelling of the emission region. At this stage, you can convert the `On-Off` dataset to a `MapDataset` and fit it using cash statistics by calling `stacked_on_off.to_map_dataset()`. A background model will be generated as `counts_off * alpha`. \n",
+    "\n",
+    "Alternatively, we show here a very basic example of modeling and fitting `On-off` datasets - using real background events instead of a background model - here, the `wstat` statistics is internally used.\n",
+    "\n",
```

Comment:
Real background events. I think this is a bit misleading stated like this. 
I would write it: using a background measurement (OFF measurement), here obtained with a ring background technique.